autogen

```diff
@@ -366,27 +382,39 @@ function autogenForFile() {
       # "php script.php" .
       #
       # Note: PHP accepts C, C++, and shell-style comments.
-      echo "<?php"
+      if [ ${ADD_RUNLINE} -eq 1 ]; then
+       echo "<?php"
+      fi
       printLicenseNonHashComment "//"
       printFileCommentTemplate "//"
-      echo
-      # E_STRICT was added in PHP 5.0 and became included in E_ALL in PHP 6.0 .
-      echo "error_reporting(E_ALL | E_STRICT);"
-      echo "?>"
+      if [ ${ADD_RUNLINE} -eq 1 ]; then
+        echo
+        # E_STRICT was added in PHP 5.0 and became included in E_ALL in PHP 6.0 .
+        echo "error_reporting(E_ALL | E_STRICT);"
```

Comment:
Maybe this is a no-code feature rather than a no-runline?

Reply:
Agreed